pkg/cmd/list.go

```diff
@@ -281,6 +259,13 @@ func (w *WhoCan) Check() (roleBindings []rbac.RoleBinding, clusterRoleBindings [
 		return
 	}
 
+	w.gr, err = w.resourceResolver.Resolve(w.verb, w.resource, w.subResource)
+	if err != nil {
+		err = fmt.Errorf("resolving resource: %v", err)
+		return
+	}
+	glog.V(3).Infof("Resolved resource `%s`", w.resource)
```

Comment:
Even if `w.resource` is empty, this message will be shown, right? The current implementation doesn't display it.
https://github.com/aquasecurity/kubectl-who-can/pull/62/files#diff-7f1690f58d7299440bd878e1c25d001bL188

Is it OK?

Reply:
> Even if `w.resource` is empty, this message will be shown, right? The current implementation doesn't display it.
> https://github.com/aquasecurity/kubectl-who-can/pull/62/files#diff-7f1690f58d7299440bd878e1c25d001bL188
> 
> Is it OK?

Good catch. Thank you. It doesn't harm, but I reverted it anyway to only invoke ResourceResolver for Kubernetes resources. It doesn't make sense to call resolver for NonResourceURL args.